clearwater-memcached/usr/share/clearwater/infrastructure/scripts/memcached

```diff
@@ -25,4 +25,6 @@ sed -e 's/^-l .*$/-l '$listen_address'/g'\
     -e 's/^# *-v *$/-v/g'\
     -e 's/^\(# *\|\)-c.*$/-c 4096/g' </etc/memcached.conf >/etc/memcached_11211.conf
 
+echo "\n# Disable listening on UDP\n-U 0" >> /etc/memcached_11211.conf
+
```

Comment:
A quick Google suggests that [echo is inconsistent](https://stackoverflow.com/questions/8467424/echo-newline-in-bash-prints-literal-n) when it comes to newlines. The same article suggests alternatives including printf, so I'll look into that and make changes as appropriate.